keps/sig-node/3960-pod-lifecycle-sleep-action/README.md

```diff
@@ -331,8 +337,21 @@ The change is opt-in, it doesn't impact already running workloads.
 
 ###### What specific metrics should inform a rollback?
 
+Metric `sleep_action_terminated_early_total` will be added in beta. 
+If it increases unreasonably, then user should check if something goes wrong and may need a rollback.
+
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
+This is an opt-in feature, and it does not change any default behavior. We manually tested enabling and disabling this feature by changing kubelet and kube-api-server config and restarting them.
+
+The manual test steps are as following:
```

Comment:
Is there an e2e job or plan to add one that covers this?

Reply:
The comments for PRR question usually make it sound like doing e2e tests for upgrades is quite difficult in upstream.

I believe this is manually and I'm not entirely sure if there is a way to automate in upstream.

Reply:
> Is there an e2e job or plan to add one that covers this?

Nope, we'll test this manually on our local cluster. e2e jobs will only run with feature-gate enabled.